src/services/fileServices/MdPageServices/MediaFileService.js

```diff
@@ -120,6 +123,12 @@ class MediaFileService {
       )
     }
 
+    if (!ALLOWED_FILE_EXTENSIONS.includes(oldExt)) {
```

Comment:
+1 for extra validation! 
Do you think we could also do a similar parsing validation in the backend of unallowed characters? (similar to what we already do in our frontend?) 

I do think this this is a 'nice to have' rather than a 'must have', feel free to resolve this comment if you feel this is not needed!

Reply:
this is already done by `this.mediaNameChecks`